javascript/react-js/advanced-concepts.md

```diff
@@ -36,15 +36,19 @@ If you start writing larger applications you will, with certainty, encounter the
 
 You might have already heard about Redux. [Redux](https://react-redux.js.org) is the most popular state management system out there. It is not a part of React, but the two can be very easily combined. Together, they make up an extremely powerful duo. The purpose is to store your application's state in a single place, commonly called a "store". You then dispatch actions to the store, where a reducer will handle the state changes. The primary benefit of using a state management library is to prevent having to pass props through multiple levels of the component tree. A state management library is often only recommended for larger applications.
 
-#### <span id="higher-order-components">4. Higher-order Components</span>
+#### <span id="context-api">4. Context API</span>
```

Comment:
Thanks for adding a `span` here. This reminds me that we should have a knowledge check link for Context API.

Can you add that as well?

Reply:
Absolutely :)